utils/words.js

```diff
@@ -45,14 +48,17 @@ const WORD_LISTS = {
     "cowpie",
     "crappy",
     "doodoo",
+    "dookie",
     "egesta",
     "faeces",
     "jobbie",
     "manure",
     "ordure",
     "sewage",
     "shitty",
+    "sludge",
     "stools",
+    "turdle",
```

Comment:
Good to have for when Turdle.app takes the world by storm and enters the common lexicon

Reply:
Good thinking